docs/configuration.md

```diff
@@ -1233,6 +1246,9 @@ Apart from these, the following properties are also available, and may be useful
   <td>
     If true, Spark will attempt to use off-heap memory for certain operations. If off-heap memory 
     use is enabled, then <code>spark.memory.offHeap.size</code> must be positive.
+    <em>Note:</em> If off-heap memory use is enabled or off-heap memory size is increased, 
```

Comment:
What is 'off-heap memory size is increased' referring to?

Reply:
increase the size of off-heap memory by `spark.memory.offHeap.size`.

Reply:
This is implied by enabling off-heap memory; I'd just remove the second part of this statement.
Below: remove the word 'recommend'. Space before the paren and fix "e.g."
But isn't this already accounted for by `spark.memory.offHeap.size`?

Reply:
OK, I have adjusted the documentation according to your reminder.
When Spark running in cluster mode (e.g. YARN), the size `spark.memory.offHeap.size` is not accounted in total memory size of container. As size of off-heap memory grows, increase the size of the non-heap memory manually.